cloudbuild.yaml

```diff
@@ -152,19 +154,12 @@ steps:
   - name: 'go-vol'
     path: '/go'
 
-- id: 'Test: Delete Cluster'
+# Disable ci-reap-clusters for now. TODO: refactor it to ci-reap-namespace
+- id: 'Test: Delete Open Match'
   name: 'gcr.io/$PROJECT_ID/open-match-build'
-  args: ['make', 'SHORT_SHA=${SHORT_SHA}', 'GCLOUD_EXTRA_FLAGS=--async', 'GCP_PROJECT_ID=${PROJECT_ID}', 'ci-reap-clusters', 'delete-gke-cluster']
+  args: ['make', 'SHORT_SHA=${SHORT_SHA}', 'GCLOUD_EXTRA_FLAGS=--async', 'GCP_PROJECT_ID=${PROJECT_ID}', 'ci-reap-clusters', 'delete-chart']
```

Comment:
Does the reaper know how to clean up namespaces yet?  What happens if a ci fails in the middle, does it leave instances of open match laying around?

Reply:
This is an incoming PR for it. I will push it once this PR get approved.

Reply:
ack